WcaOnRails/config/database.yml.travis

```diff
@@ -0,0 +1,9 @@
+test:
```

Comment:
Hmm... I think the DATABASE_URL environment variable is a better way of
providing the developer flexibility you're describing. I generally dislike
.sample files, as they're an extra step when setting up an initial
development environment.

I feel like it would be cleaner to use erb inside the .yml file to have
different settings for Travis. Copying a file around seems hacky to me.
What do you think?

On Sat, Sep 19, 2015, 18:51 Tim Habermaas <notifications@github.com> wrote:

> In WcaOnRails/config/database.yml.travis
> <https://github.com/cubing/worldcubeassociation.org/pull/173#discussion_r39925930>
> :
>
> > @@ -0,0 +1,9 @@
> > +test:
>
> These two files being identical is just a coincidence. I don't want travis
> to break if we change the local connection settings. Speaking of local
> settings: I usually don't commit database.yml to the repository, but
> provide a database.yml.sample since people might want to customize their
> database settings which gets annoying if the file you modify is checked
> into source control. I decided against removing it, though since it would
> probably break the vagrant box.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/173/files#r39925930>
> .
>